### PR TITLE
Update the eviction server to not set stuck if it isn't full.

### PR DESCRIPTION
**Pull request by agorrod**
When only looking for pages to force out, the cache isn't really stuck.
The trouble is that if we set stuck the eviction server doesn't clear walks
which can lead to it always holding a reference t othe page we (really) want
to evict.

Refs: #1777

### CODE REVIEW

**Comment by agorrod** [comment]
@michaelcahill Could you review this one please?

File: bench/wtperf/runners/parallel-pop-stress.wtperf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# wtperf options file: Run populate multi-threaded with a btree. This can
+# uncover conditions where pages are too busy for eviction to work as
+# expected
+conn_config="cache_size=1GB"
+table_config="type=file"
+icount=10000000
+report_interval=5
+populate_threads=20
+run_time=0
```
Comment (michaelcahill) [inline bench/wtperf/runners/parallel-pop-stress.wtperf]: Should we run this as part of a jenkins job?

**Comment by michaelcahill** [comment]
Thanks, @agorrod -- lgtm and I'm pleased you've opened #1788.  I'll merge?

File: bench/wtperf/runners/parallel-pop-stress.wtperf (hunk 1 above)
Comment (agorrod) [inline bench/wtperf/runners/parallel-pop-stress.wtperf]: That was my plan. I've created a template here:
http://build.wiredtiger.com:8080/job/wiredtiger-perf-parallel-populate-stress/

**Pull request merged**